api/models/release.go

```diff
@@ -18,7 +19,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/convox/rack/api/crypt"
 	"github.com/convox/rack/api/structs"
-	"github.com/convox/rack/manifest"
+	mp "github.com/convox/rack/manifest"
```

Comment:
So let's go the other way with this to avoid surprises around the namespace. Perhaps name the variable `m` down in the function and let this be `manifest`